### ISSUE TITLE
SimpleTask calendar disappears after a while

### ISSUE THREAD

**Issue by ddisanti** [mpcjanssen/simpletask-android#672]
Cloudless 8.3.0 from Play Store.
I set reminders for due dates on the same day at 9:30. But I do not receive notifications. When opening the calendar app, SimpleTask calendar has disappeared. Then I open SimpleTask, I edit any task, save it and close SimpleTask. Only then Simpletask calendar shows up again.
It happens if I don't open Simpletask for a while, lets say the whole afternoon.

**Comment by smichel17** [comment]
Debug log would be very helpful here, so I guess we're blocked on #670

**Comment by smichel17** [comment]
@ddisanti

**Comment by ddisanti** [comment]
It happened again today. I opened ST for last time yesterday at afternoon. I was about to open it early today and remember this issue. So, I opened the Google Calendar app and Simpletask calendar wasn't there.
I have set ST calendar reminders at 07:30AM, but this issue/bug make that pointless.
Debug log attached.
[Simpletask-20170213.txt](https://github.com/mpcjanssen/simpletask-android/files/771031/Simpletask-20170213.txt)

**Comment by mpcjanssen** [comment]
@ddisanti , will restarting the application (e.g. really killing it and restarting) recreate the calendar?

**Comment by ddisanti** [comment]
No needed to kill ST. Just exit with BACK button and the calendar is
recreated.

**Comment by ddisanti** [comment]
@mpcjanssen I've been uisng the build you posted [here](https://gitter.im/mpcjanssen/simpletask-android?at=58a1dc1700c00c3d4f329445)
I made a list for testing and added some task with due and threshold dates  setted . Then I've closed ST-debug and didn't opened it for a day.
The notifications worked fine. They were triggered as expected.
The only thing is that during this test the ST calendar was missing from the Google Calendar app and it was recreated only after I opened ST.
In short, notifications worked ok but ST calendar dissappear untill ST is started

**Comment by ddisanti** [comment]
@mpcjanssen it seems to be fixed in latest Cloudless Beta 9.0.0. 
Notifications are properly triggered and ST calendar is there.

**Comment by mpcjanssen** [comment]
@ddisanti because I don't understand exactly what was causing this issue, please keep monitoring this for a while before closing this issue.

**Comment by vojtechkral** [comment]
Huh, weird. As far as I know, there wasn't any code change between those builds. Android calendar works in mysterious ways...

**Comment by mpcjanssen** [comment]
I did change some calendar sync stuff recently (mostly using config settings directly) which could make a difference.
